docs/source/usersguide/materials.rst

```diff
@@ -55,6 +55,20 @@ following would add 3.2% enriched uranium to a material::
 In addition to U235 and U238, concentrations of U234 and U236 will be present
 and are determined through a correlation based on measured data.
 
+It is also possible to perform enrichment of any element that is composed
+of two naturally-occurring isotopes (e.g. Li, B) in terms of atomic percent.
+To invoke it, provide additional arguments `enrichment_target` to
```

Comment:
I suggest a slight rewording of this line:
"To invoke this, provide the additional argument `enrichment_target` to".